tests/test_logic.py

```diff
@@ -479,3 +475,22 @@ def test_infinity_sign_errors(func):
     out = dpnp.empty_like(x, dtype="int32")
     with pytest.raises(ValueError):
         getattr(dpnp, func)(x, out=out)
+
+
+@pytest.mark.parametrize("dtype", get_all_dtypes(no_bool=True, no_complex=True))
+@pytest.mark.parametrize(
+    "rtol", [1e-05, dpnp.array(1e-05), dpnp.full(10, 1e-05)]
+)
+@pytest.mark.parametrize(
+    "atol", [1e-08, dpnp.array(1e-08), dpnp.full(10, 1e-08)]
+)
+def test_isclose(dtype, rtol, atol):
```

Comment:
needs to uncomments tests in `test_sycl_queue.py` and `test_usm_type.py` for `isclose`